packages/file_selector/file_selector/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## NEXT
+## 0.9.2+1
 
-* Updates minimum Flutter version to 2.10.
```

Comment:
For future PRs: please don't delete things from CHANGELOGs.

This will get a changelog exemption too though, so you can just revert this whole file.

Reply:
Thanks for the feedback, it was our mistake. Just to double check, we're not going to modify this file, neither bump to a new version.